### PR TITLE
checkstyle HideUtilityClassConstructor

### PR DESCRIPTION
**Pull request by garyrussell**
https://sonar.spring.io/issues/search#componentRoots=org.springframework.integration%3Aspring-integration|createdAt=2016-04-01T10%3A00%3A36%2B0000|sort=UPDATE_DATE|asc=false

### CODE REVIEW

**Comment by artembilan** [comment]
Ha! Great! I thought recently to do this :smile: 
Thank you for taking care!

**Comment by garyrussell** [comment]
I get pinged by Sonar with new violations; so that's what triggered this.

File: spring-integration-amqp/src/main/java/org/springframework/integration/amqp/support/MappingUtils.java [hunk 1]
```diff
@@ -32,6 +32,10 @@
  */
 public class MappingUtils {
 
+	private MappingUtils() {
+		super();
```
Comment (artembilan) [inline spring-integration-amqp/src/main/java/org/springframework/integration/amqp/support/MappingUtils.java]: Would you mind to explain why do we need call `super` from all of these new `private` ctors?
Looks like that works as is even without it and `checkstyle` is happy, too.

Comment (garyrussell) [inline spring-integration-amqp/src/main/java/org/springframework/integration/amqp/support/MappingUtils.java]: There's a Sonar rule that doesn't like empty blocks; I could have added a comment just to keep sonar happy, but this doesn't hurt - the bytecode is identical:

`INVOKESPECIAL java/lang/Object.<init> ()V`

with or without the `super()`.

Comment (artembilan) [inline spring-integration-amqp/src/main/java/org/springframework/integration/amqp/support/MappingUtils.java]: OK. That's great!
Merging...

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-integration/commit/43af472c3a9b57594dae3fb06d4173708de9be94

**Pull request closed without merging**